src/modules/misc/_loading.scss

```diff
@@ -46,3 +46,9 @@
         }
     }
 }
+
+.loadingSpinner-3YlU8g {
+    display: flex!important;
```

Comment:
Need to add a short comment explaining `!important`s. See [Comments & Debugging](https://github.com/TakosThings/Fluent-Discord/blob/develop/.github/CONTRIBUTING.md#comments--debugging) in contributing.md